packages/components/src/styles/db-ui-components.scss

```
@@ -70,3 +73,9 @@ db-alert,
 dbalert {
 	inline-size: inherit;
 }
+
+// for db-badge
+
+*:has(.db-badge[data-placement^="corner"]) {
```

```suggestion
:has(.db-badge[data-placement^="corner"]) {
```

Just to prevent an asterisk for once (as it should be optional in this case).

---

fixed